Adds Cluster Station

CawsForConcern opened the pull request:
<!-- IT'S NOT WIZDENS REPO, IF YOU WANT TO ADD YOUR CHANGES ON ALL SERVERS, CREATE PR TO WIZDENS REPO -->

## Short description
<!-- What do you propose to change with your PR? -->
Adds a nostalgic low pop station (0-35 players), loving remade tile-by-tile. This is a _very old_ station that Pig wanted to see brought back for modern Space Station.

Most of the original layout has been retained, but Security has been completely revamped for Gen Pop, turrets controls, Brigmedic, etc. Cluster's emergency shuttle has also been updated to have proper distro and generally fixed up.

## Why we need to add this
<!-- What is the reason for adding these changes? Please post links to Discussions as well as Bug Reports here. Please describe how this will change the game balance. -->
Map variety

## Media (Video/Screenshots)
<!--
If your PR contains in-game changes you must provide screenshots/videos of the changes.
-->
<img width="3328" height="4256" alt="StarlightCluster-0" src="https://github.com/user-attachments/assets/38021e4e-f1ce-46ef-914e-012a3f55219c" />


## Checks
<!-- check boxes for faster reviewing of your PR -->

- [x] I do not require assistance to complete the PR.
- [x] Before posting/requesting review of a PR, I have verified that the changes work.
- [x] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [x] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**
<!--
If you want the players to know about changes made in this PR, specify them using the template outside the comment. Short and informative.
-->
:cl: CawsForConcern
- add: Added Cluster Station

CawsForConcern committed:
- Bar
- Botany
- Medical Department
- Security Department

CawsForConcern committed:
- Cargo Department
- Engineering Department
- Janitor

CawsForConcern committed:
- Map prototype (jobs to be adjusted later)
- Chapel
- Library
- Gravity Generator Room
- Maints Surgery (replaces old Surgery)
- Disposals Room
- Science Anom Generator Room
- Boxing Ring
- Maints Bar
- Space Bar
- South Solars
- Cult Arcade
- Vigilante Room
- about a quarter of the maints hallways are done

CawsForConcern committed:
- Science Department
- Mime / Clown / Musician
- Tech Vault
- Cyrosleep
- Even more common areas
- Even more maints

CawsForConcern committed:
- Bridge
- BSO
- Captain's Room
- Conference Room
- Evac
- Courtroom
- Spare Docking Arm
- AI
- Basically the rest of the Maints
- Interrogation
- North Solars

CawsForConcern committed:
Invalid devices removed

CawsForConcern committed:
- Shooting Range
- Distro
- Waste
- Air Alarm Linking
- Fire Alarm Linking

CawsForConcern committed:
A few distro fixes
Power network first pass

CawsForConcern committed:
- the serializer didn't like saving the thrown spent shellcasings at the SEC firing range, removed them
- fixed a bunch of small power issues
- named all of the air alarms (hhh)
- named all of the fire alarms (HHHHHHHHHH)

CawsForConcern committed:
- station beacons

CawsForConcern committed:
- holopads

CawsForConcern committed:
fixgridatmos + fixrotations

CawsForConcern committed:
- added an updated emergency shuttle for Cluster
- added nitrogen lockers to the emergency shuttle
- added actual distro to the emergency shuttle
- fixed a thruster that was permanently turned off
- fixed the emergency shuttle's bridge lockdown button so it's acutally visible (was previously completely hidden under a screen)

CawsForConcern committed:
- disposal routing

CawsForConcern committed:
- updated job counts

PubliclyExecutedPig commented:
4 minor nitpicks ive noticed in order from most pressing to least pressing
> Roundstart plasma solars
> No medical bed in brigmed
> White tiles in kitchen should be plastic instead of steel (very minor)
> I forgot the fourth one so its probably not important

Very faithful remake, I'm a big fan!

PubliclyExecutedPig commented:
I remembered the fourth one, AI core needs intercoms

ScorchFollower commented:
Definitely one of the more interestin' maps I've played on when I first started playing SS14. I dig this blast from the past. <3

CawsForConcern committed:
- feedback: removed roundstart plasma solar panels

CawsForConcern committed:
- feedback: made the medical bed easier to see by giving it a medical bedsheet (instead of being cheeky with brigmedic's bedsheet)

CawsForConcern committed:
- feedback: tiles in kitchen are now plastic (instead of steel)

CawsForConcern committed:
- feedback: intercoms for AI core (four of them around the walls surrounding AI core)

CawsForConcern committed:
- forgot to put in a station anchor so I did that, lives where the pirate room was

CawsForConcern commented:
> 4 minor nitpicks ive noticed in order from most pressing to least pressing
> 
> > Roundstart plasma solars
> > No medical bed in brigmed
> > White tiles in kitchen should be plastic instead of steel (very minor)
> > I forgot the fourth one so its probably not important
> 
> Very faithful remake, I'm a big fan!

- Solars de-plasmafied
<img width="698" height="704" alt="image" src="https://github.com/user-attachments/assets/f1192d2e-bdb7-4d69-961f-e278da42cc69" />

- Changed the brigmedic bedsheet to a medical bedsheet to make it easier to see it's a medical bed
<img width="492" height="418" alt="image" src="https://github.com/user-attachments/assets/0aba28fc-b6d0-4b4b-9ff0-5b6e277f7ca4" />

- Tiles made plastic in kitchen (no picture because they look identical)

- Intercoms for AI Core
<img width="348" height="357" alt="image" src="https://github.com/user-attachments/assets/34d78745-725d-4d8c-90c1-c6c2790dc9cb" />

One more thing, I forgot to put in a Station Anchor so I did that. It's where the pirate gear room was, I moved the pirate gear to the maints hall.
<img width="795" height="637" alt="image" src="https://github.com/user-attachments/assets/b7292374-4b72-4d22-9407-8b3fef45bcdc" />

PubliclyExecutedPig reviewed (approved):
😁

CawsForConcern committed:
- forgot to name the Fax Machines, fixed

CawsForConcern committed:
- spawnpoints QA pass

ShadesMars committed:
Merge branch 'starlight-dev' into ClusterStation

ShadesMars reviewed (changes requested):
Test fails Valid

CawsForConcern committed:
- added StarlightCluster to map pool, should fix test fail

CawsForConcern committed:
Merge branch 'ClusterStation' of github.com:CawsForConcern/space-station-14 into ClusterStation

CawsForConcern committed:
added ClusterStation to map tests, oop

CawsForConcern committed:
Merge branch 'starlight-dev' into ClusterStation

CawsForConcern commented:
> Test fails Valid

I thiiiiiink I fixed it (hopefully!)
